### PR TITLE
remaining sections mobile first portfolio

### PR DESCRIPTION
**Pull request by RoyJumah**
I added the work section that included an image of the project and the languages that I used
I added a border between the different sections of the page.
I added an about myself section containing my social media links
I added a border radius to the hero section and the about us section
I added a background color to the body that distinguished it from the rest of the sections.
I added a section containing my skills, frameworks and languages

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -100,3 +116,167 @@ body {
...
+
+/* FLEX */
+.works {
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
+  align-items: center;
+  justify-content: center;
+  padding: 1.6rem;
+}
```
Comment (levy002) [inline style.css:135]: - [ ] Kindly make sure to use `display: grid` on the works section container in order to meet the project's requirements. :wink:

File: Index.html [hunk 2]
```diff
@@ -83,6 +82,222 @@ <h2 class="heading-secondary">Let's Connect</h2>
...
+            <h2 class="heading-tertiary">Multi-post stories</h2>
+            <ul class="portfolio-list">
+              <li class="name">Canopy</li>
+              <li class="dev">Back End Dev</li>
+              <li class="year">2015</li>
+            </ul>
+            <div class="portfolio-info">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </div>
```
Comment (levy002) [inline Index.html:193]: - [ ] Kindly make sure the project description paragraph is wrapped into the `p` tag. :wink:  Please implement these changes to all project cards.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @RoyJumah   , :wave:
### I'm Levy, your PR reviewer

Excellent work :sunglasses: However, there are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: Index.html (hunk 2 above)
Comment (RoyJumah) [reply Index.html:193]: > * [ ]  Kindly make sure the project description paragraph is wrapped into the `p` tag. 😉  Please implement these changes to all project cards.

Thanks, working on it

**Review by TuyishimireEric — APPROVED** [review]
# Status: APPROVED  :+1: :+1:  
## Hi @michaelamponsah   :hand: 

Thank you for Implementing the requested changes from the previous code reviewer :clap: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## To Highlight
- All tests are passing ✔️ 
- Correct use of GitFlow ✔️
- Professional PR ✔️
- Professional commit  ✔️

Image


## Optional suggestions

- [ ] [OPTIONAL] I noticed that the MIT link is broken, so I suggest adding the MIT.md file to your project 👌,
image

Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉
```The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯```

- [ ] [OPTIONAL] It would be better also if you provide a valid LinkedIn link to your profile for the best user experience

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**